_tests/unit_tests/modules/footer-button-spec.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+var chai = require( 'chai' );
+var sinon = require( 'sinon' );
+var expect = chai.expect;
+var jsdom = require( 'mocha-jsdom' );
+
+describe( 'The Footer Return-to-Top Button', function() {
+
+  var $;
+
+  jsdom();
+
+  before( function() {
+    $ = require( 'jquery' );
+
+  } );
+
+  beforeEach( function() {
+    // Adding a simplified version of the thing we want to test.
+    // Then calling the jQuery to test after
+    $( 'body' ).html( $( '<a class="js-return-to-top"' +
```

Comment:
Try to not to have that folder for as long as possible. We want small simple tests.